packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -184,7 +184,7 @@ const DropdownList = React.forwardRef<HTMLDivElement, TDropdownList>((props, lis
                     role='list'
                     ref={list_ref}
                 >
-                    <ThemedScrollbars height={`${list_dimensions[1]}px` || '200px'}>
+                    <ThemedScrollbars height={list_dimensions[1] || '200px'}>
```

Comment:
@niloofar-deriv, don't we need the unit (`px`) for `list_dimensions[1]` content?

Reply:
and we didn't use px in the master